GoogleForCreators/web-stories-wp
# Story Player: Use intrinsic AMP layout mode

## Summary

As documented in #5562 a solution that worked well. 

## Relevant Technical Choices



## To-do



## User-facing changes

Story embeds should have the same size on AMP and non-AMP.

## Testing Instructions

1. Embed a story in  a blog post
2. View blog post and notice size of story.
3. Enable AMP
4. View blog post with  AMP enabled and verify that the size is the same

---



Fixes #5562
